application-task-ui/src/main/resources/TaskManager/TaskManagerLiveTableResults.xml

```diff
@@ -36,5 +36,34 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{include reference="XWiki.LiveTableResults" /}}</content>
+  <content>{{include reference="XWiki.LiveTableResultsMacros" /}}
+
+{{velocity wiki="false"}}
+## the $extra variable is passed to the call to #gridresultwithfilter below and is used to perform some default
+## filtering on the LiveTable data even when the user has not performed any column filtering yet.
+## When users start using column filtering, the generated DB query will combine both the default filtering and what the
+## user has entered in the column filters.
+#set ($extra = "and doc.fullName not like '%Template'")
+#set ($params = {})
+#if ("$!request.space" != '')
+#set ($extra = "${extra} AND doc.space = :doc_space")
```

Comment:
missing formatting? or does it add empty lines on the response? not sure it does